src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageParser.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.microsoft.azure.servicebus.MessageBody;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+import java.io.IOException;
+import java.util.List;
+
+public class PaymentMessageParser {
+
+    private static final ObjectMapper objectMapper;
+
+    static {
+        objectMapper = new ObjectMapper();
```

Comment:
This object mapper should come from global configuration, where it would have common setup for serialisation/deserialisation.

Reply:
global mapper  configuration might have different configurations ,as this is just to read from queue may be it is better not use global one ?

Reply:
We probably want one way of serialising/deserialising data when talking to outside world. Otherwise, it would be hard to manage differences. If, however, that had to happen (extremely unlikely), we could just create another bean.